src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -125,74 +124,59 @@ def acr_show(registry_name, resource_group_name=None):
 
     return client.get_properties(resource_group_name, registry_name)
 
-def acr_update_set(registry_name,
-                   resource_group_name=None,
-                   parameters=None):
-    registry, resource_group_name = get_registry_by_name(registry_name, resource_group_name)
-
-    admin_user_enabled = parameters.admin_user_enabled if parameters.admin_user_enabled else False
-
-    # Set storage account
-    storage_account = None
-    if parameters.storage_account_name:
-        storage_account_key = get_access_key_by_storage_account_name(parameters.storage_account_name)
-        storage_account = StorageAccountProperties(
-            parameters.storage_account_name,
-            storage_account_key
-        )
-
-    # Set tags
-    tags = parameters.tags #pylint: disable=no-member
-    if not isinstance(tags, dict):
-        tags = {}
+def acr_update_get(client,
+                      registry_name,
```

Comment:
By the way, it seems that some other modules fail in the linter (acs, etc.) which is out of the scope of this PR. Any thoughts?